test/e2e/validation_test.go

```diff
@@ -464,6 +465,68 @@ func testExpectedServicesRunning(t *testing.T) {
 	}
 }
 
+// testServicesConfigMap tests 2 aspects of expected functionality for the services ConfigMap
+// 1. It exists on operator startup and 2. It is recreated when deleted
+func testServicesConfigMap(t *testing.T) {
+	tc, err := NewTestContext()
+	require.NoError(t, err)
+
+	operatorVersion, err := getWMCOVersion()
+	require.NoError(t, err)
+	// There is no easy way to populate ldflags when running e2e tests so that version.Get() will return
```

Comment:
As a complete e2e test, can you fetch all ConfigMaps where the name starts with `windows-services` and validates that there is one, and only one,  that matches `"windows-services"  + "-" + operatorVersion`, then use the content for further validation.

Reply:
Since we don't delete any misnamed configmaps yet, I don't think fetching all `windows-services-*` CMs is needed. Your suggestion is good to keep in mind when we do delete outdated configmaps/handle upgrades in coming stories.